bootstrap/bootstrap-storage.sh

```diff
@@ -41,60 +40,57 @@ else
   JAVA="${JAVA_HOME}/bin/java"
 fi
 
-TABLE_INITIALIZER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.TablesInitializer
-for file in "${BOOTSTRAP_DIR}"/lib/*.jar;
+if grep -E "^(\s)*db.type:(\s)*\"mysql\"(\s)*$" $CONFIG_FILE_PATH > /dev/null; then
+  if [ ! -f ../libs/$MYSQL_JAR_PATTERN ] ; then
```

Comment:
1. This assumes you are inside bootstrap dir which may not be the case. 

Use `${BOOTSTRAP_DIR}/../libs/` 

2. This will fail if there are multiple files matching the glob